apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/AllTypesTableFixture.cs

```diff
@@ -28,6 +28,91 @@ public AllTypesTableFixture() : base("TypesTable")
         {
         }
 
+        /// <summary>
+        /// Creates INSERT command for the table created after executing the <see cref="CreateTable"/> method execution.
+        /// </summary>
+        /// <returns>The DML command to insert data into a table.</returns>
+        public string CreateInsertCommand()
+        {
+            // The emulator doesn't yet support the JSON type.
```

Comment:
Couldn't this be rather simpler with an interpolated verbatim string literal which contains something like:

```csharp
{ (RunningOnEmulator ? "" : "JsonArrayValue,") }
```

and

```csharp
{ (RunningOnEmulator ? "" : "@JsonArrayValue,") }
```

(You'd want to reorder it so that the JSON array value isn't the last element, unless Spanner is happy with trailing commas.)

Reply:
(There may be other places we could do that, too.)

Reply:
Note that now this is the only statement, the method can be turned into an expression-bodied one.